packages/api/src/schema/query.ts

```diff
@@ -474,6 +474,19 @@ export default [
       Determines if a token is valid/active for email verification
       """
       validateVerifyEmailToken(token: String!): Boolean!
+
+      """
+      Returns aggregated report of all the events that occurred in the system
+      """
+      eventStatistics(types: [String!], from: Timestamp, to: Timestamp): [EventStatistics]!
```

Comment:
The case [Entity]! is almost never needed because it would include arrrays like this: [null, undefined]. Only in very rare cases we want this potential outcome.

99% of the time use [Entity!]! for array like return types in graphql